awx/api/permissions.py

```diff
@@ -43,7 +46,10 @@ def check_get_permissions(self, request, view, obj=None):
         return check_user_access(request.user, view.model, 'read', obj)
 
     def check_post_permissions(self, request, view, obj=None):
-        if hasattr(view, 'parent_model'):
+        if getattr(view, 'IS_COPY_VIEW', False):
+            # Delay copy permission check for later in post() method.
+            return True
```

Comment:
I don't think there is any problem with what you're doing, but I would rather you just change the permission_class of `CopyAPIView` to a no-op permissions class.

Reply:
Good point 👍